src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Microsoft.CodeAnalysis.Razor.Workspaces.csproj

```diff
@@ -6,6 +6,7 @@
     <EnableApiCheck>false</EnableApiCheck>
     <IsShippingPackage>false</IsShippingPackage>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <LangVersion>Latest</LangVersion>
```

Comment:
Do we want `Latest` of `9.0`?

Reply:
We've been marking Latest elsewhere so I just followed suit